src/openms/source/ANALYSIS/ID/AScore.cpp

```diff
@@ -187,6 +203,17 @@ namespace OpenMS
     return base_match_probability;
   }
 
+  /**
```

Comment:
We usually put the doc in the header file… not sure if it will show up in documentation this way

Reply:
@timosachsenberg a question here. The entire purpose of the docs is to make it closer to the code for developers to understand it. Why in C++ it is in the `.h` files.